src/components/tooltip/tooltip-content.tsx

```diff
@@ -25,10 +25,31 @@ export class TooltipContent {
     @Prop({ reflect: true })
     helperLabel: string;
 
+    /**
+     * The maximum amount of characters before rendering 'label' and 'helperLabel' in two rows.
+     * Defaults to 50 characters.
+     */
```

Comment:
@jgroth is it OK to repeat the same documentation in two places? We say this one time here, and one time in the main component.

Reply:
`@inheritdoc` won't do anything here since we are not inheriting from a parent. This component is marked as `@private`  however so it won't show up in the docs, I don't think documentation here is as important

I can't remember why we needed to have two components like this though :thinking:

Reply:
> why we needed to have two components like this though

because of the Portal i guess?

> I don't think documentation here is as important
I suggest to remove them and just say: `read more in looltip.tsx` or `refer to main component` or something like that. We don't want to do a lot of bookkeeping.